dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -277,6 +249,9 @@ String retrieveUsingRuleTemplate(String id) throws RuleNotFoundException, IOExce
                          row.getString(CONDITION_COL), row.getString(ACTION_COL));
   }
 
+  /**
+   *
```

Comment:
should have some actual content

Reply:
I meant to remove that for this commit, but I now have added the javadoc.